docs/styleguide/scripts/main.js

```diff
@@ -1,5 +1,13 @@
-/**
- * Created by d063436 on 5/21/17.
- */
-console.log('Just for js test purpose');
 
+$(document).ready(function(){
+    var primaryNavigationDropdown = $('.yaas-signature-nav__list');
+    $('.primary-nav--toggle').on('click', function(){
+        $(this).toggleClass('active');
+        $(primaryNavigationDropdown).toggleClass('open');
+    });
+
+    $('#menu-toggle').on('click', function(){
```

Comment:
You may not need this at all, it looks like both of these share the exact same behavior. In fact, this same script could be used for lots of toggle behaviors.